var/spack/repos/builtin/packages/py-scine-chemoton/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyScineChemoton(PythonPackage):
+    """Software driving the automated exploration of chemical reaction networks"""
+
+    homepage = "https://scine.ethz.ch/download/chemoton"
+    pypi = "scine_chemoton/scine_chemoton-2.2.0.tar.gz"
+    git = "https://github.com/qcscine/puffin.git"
+
+    version("develop", branch="master")
```

Comment:
I would just call the version master to match the github

Reply:
It is not very consistent in spack, I found several packages which call the latest version "develop" regardless of branch name, but also a lot of packages use "main" or "master" depending on the actual branch name. I'm fine with either way, but found the former strategy slightly better because the git version doesn't change the name if upstream renames from "master" to "main".

Reply:
That's for historical reasons. Once upon a time (several years ago) develop was the only version that meant "newer than all stable releases". Other versions like main or master were considered to be older than all stable releases, and patches wouldn't apply correctly. This was fixed a while back but there's still a lot of packages that haven't been updated. See https://github.com/spack/spack/blob/v0.19.0/lib/spack/spack/version.py#L64 for the full list of versions that are accepted.

Reply:
Thanks for the clarification, changed accordingly.

Reply:
It still says develop